From: intrigeri <intrigeri@boum.org>
Date: Mon, 10 Sep 2018 09:41:49 +0000
Subject: AppArmor: give Web Content processes read access to the startup
 cache, otherwise they fail to load

(cherry picked from commit eb328f2abe7b681c779a6cb7e49657ac93ecd005)
---
 apparmor/torbrowser.Browser.plugin-container | 1 +
 1 file changed, 1 insertion(+)

diff --git a/apparmor/torbrowser.Browser.plugin-container b/apparmor/torbrowser.Browser.plugin-container
index ae2a9ba..7ec8a00 100644
--- a/apparmor/torbrowser.Browser.plugin-container
+++ b/apparmor/torbrowser.Browser.plugin-container
@@ -57,6 +57,7 @@ profile torbrowser_plugin_container {
   owner @{torbrowser_home_dir}/fonts/** r,
   owner @{torbrowser_home_dir}/omni.ja r,
   owner @{torbrowser_home_dir}/TorBrowser/Data/Browser/profile.default/extensions/*.xpi r,
+  owner @{torbrowser_home_dir}/TorBrowser/Data/Browser/profile.default/startupCache/* r,
   owner @{torbrowser_home_dir}/TorBrowser/Data/Browser/profile.default/tmp/* rw,
   owner @{torbrowser_home_dir}/TorBrowser/Data/fontconfig/fonts.conf r,
   owner @{torbrowser_home_dir}/TorBrowser/Tor/ r,
